packages/atomic/tailwind.config.js

```diff
@@ -1,6 +1,7 @@
 const plugin = require('tailwindcss/plugin');
 
 module.exports = {
+  mode: 'jit',
```

Comment:
jit is enabled by default in tailwind v3 https://tailwindcss.com/docs/upgrade-guide#migrating-to-the-jit-engine
```suggestion
```